elements_imaging/imaging.py

```diff
@@ -54,22 +51,23 @@ def activate(imaging_schema_name, scan_schema_name=None, *, create_schema=True,
 
 # -------------- Functions required by the elements-imaging  --------------
 
-def get_caiman_dir(processing_task_key: dict) -> str:
+def get_imaging_root_data_dir() -> str:
     """
-    Retrieve the CaImAn output directory for a given ProcessingTask
-    :param processing_task_key: a dictionary of one ProcessingTask
-    :return: a string for full path to the resulting CaImAn output directory
+    get_imaging_root_data_dir() -> str
+        Retrieve the root data directory - e.g. containing all subject/sessions data
+        :return: a string for full path to the root data directory
     """
-    return _linking_module.get_caiman_dir(processing_task_key)
+    return _linking_module.get_imaging_root_data_dir()
 
 
-def get_suite2p_dir(processing_task_key: dict) -> str:
+def get_session_directory(session_key: dict) -> str:
```

Comment:
Should this function be defined in `workflow-imaging/workflow_imaging/paths.py` and imported in `workflow-imaging/workflow_imaging/pipeline.py`?  Additionally, I might be missing it, but I don't see where this function utilized?

Reply:
There will be some fairly major changes on the `workflow-imaging` accompanying this new `elements-imaging`, still under development, I haven't issued PR for that yet